jest.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     '_processCSS.js',
     '_processCss.js',
     '_webpack.config.js',
+    'packages/postcss-lowercase-text/src/__tests__/data/',
```

Comment:
Why it here?

Reply:
jest was running the test running in this directory as well as there is an `.js` file which is true for jest's default test file pattern. So I added them. this directory should not be tested as it has data only.